todos/src/Types/todo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// export type todo = {
+//     id: string;
+//     label: string;
+//     active: boolean;
+// }
+
+export class todo {
```

Comment:
Is making this a class the right move given what we discussed about class vs interface/type?

Reply:
This is an open question. Its ok to deterine thats its ok, but pls explain why.

Reply:
Anyways, a type should always be capital letters